src/frontends/common/src/manager.cpp

```diff
@@ -242,3 +242,40 @@ template <>
 FrontEnd::Ptr FrontEndManager::load_by_model(const std::vector<ov::Any>& variants) {
     return load_by_model_impl(variants);
 }
+namespace ov {
+namespace frontend {
+
+namespace {
+
+class FrontEndManagerHolder {
+    std::mutex m_mutex;
+    std::weak_ptr<FrontEndManager> m_manager;
```

Comment:
Is it a weak pointer intentionally? Should the FrontEndManager objects created in this pattern be able to be destroyed and then replaced with other instances if needed?

Reply:
yes, it is designed as a weak pointer so that it can be destroyed during openvino dll/lib unloaded, ov::core also take a reference of it so that it will not be released before ov::core destruction.